main/test/ca/uwaterloo/flix/language/phase/unification/TestQMCtoBoolFormula.scala

```diff
@@ -0,0 +1,270 @@
+  }
+
+  test("Minimize.Var1AndVar2AndVar3.05") {
+    assertResult(expected = And(And(Var(0), Var(1)), Var(2)))(actual = alg.minimizeQMC(And(And(And(Var(0), Var(1)), Var(2)), And(And(Var(0), Var(1)), Var(2)))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3.06") {
+    assertResult(expected = And(And(Var(0), Var(1)), Var(2)))(actual = alg.minimizeQMC(And(And(And(Var(2), Var(1)), Var(0)), And(And(Var(0), Var(1)), Var(2)))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3.07") {
+    assertResult(expected = False)(actual = alg.minimizeQMC(And(And(And(Var(2), Var(1)), Var(0)), Not(And(And(Var(0), Var(1)), Var(2))))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3.08") {
+    assertResult(expected = True)(actual = alg.minimizeQMC(Or(And(And(Var(2), Var(1)), Var(0)), Not(And(And(Var(0), Var(1)), Var(2))))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3.09") {
+    assertResult(expected = And(And(Var(0), Var(1)), Var(2)))(actual = alg.minimizeQMC(And(And(And(Var(2), Var(1)), Var(0)), Or(Var(2), And(Var(2), Var(2))))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3.10") {
+    assertResult(expected = Or(Or(Var(2), Var(0)), Var(1)))(actual = alg.minimizeQMC(Or(Var(0), Or(Var(1), Or(Var(2), Or(Var(0), Or(Var(1), Var(2))))))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3.11") {
+    assertResult(expected = True)(actual = alg.minimizeQMC(Or(Not(Var(0)), Or(Var(1), Or(Var(2), Or(Var(0), Or(Var(1), Var(2))))))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3.12") {
+    assertResult(expected = Or(Or(Var(2), Var(0)), Var(1)))(actual = alg.minimizeQMC(Or(Or(Var(2), Or(Var(1), Var(0))), And(Var(0), And(Var(0), And(Var(2), Or(Var(0), Or(Var(1), Var(2)))))))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3.13") {
+    assertResult(expected = And(Var(0), Var(2)))(actual = alg.minimizeQMC(And(Or(Var(1), Not(Var(1))), And(Var(2), Var(0)))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3.14") {
+    assertResult(expected = Var(0))(actual = alg.minimizeQMC(Or(Var(0), And(And(Var(1), Not(Var(1))), Or(Var(2), And(Var(2), Var(1)))))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3AndVar4.01") {
+    assertResult(expected = And(And(And(Var(0), Var(1)), Var(2)), Var(3)))(actual = alg.minimizeQMC(And(And(Var(0), Var(1)), And(Var(2), Var(3)))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3AndVar4.02") {
+    assertResult(expected = And(And(And(Var(0), Var(1)), Var(2)), Var(3)))(actual = alg.minimizeQMC(And(And(Var(3), Var(2)), And(Var(0), Var(1)))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3AndVar4.03") {
+    assertResult(expected = Or(Or(Or(Var(1), Var(3)), Var(0)), Var(2)))(actual = alg.minimizeQMC(Or(Or(Var(0), Var(1)), Or(Var(2), Var(3)))))
+  }
+
+  test("Minimize.Var1AndVar2AndVar3AndVar4.04") {
+    assertResult(expected = Or(Or(Or(And(Not(Var(0)), Not(Var(2))), And(Not(Var(1)), Not(Var(2)))), And(Not(Var(0)), Not(Var(3)))), And(Not(Var(1)), Not(Var(3)))))(actual = alg.minimizeQMC(Not(Or(And(Not(Not(Var(0))), Not(Not(Var(1)))), And(Not(Not(Var(2))), Not(Not(Var(3))))))))
+  }
```

Comment:
These are just the tests copied from TestBoolFormulaTable. Would you like a few larger tests as well?

Reply:
I a few manually written tests would be useful, yes.